mlir/lib/Conversion/TosaToLinalg/TosaToLinalg.cpp

```diff
@@ -1444,105 +1447,6 @@ class ResizeUnaryConverter : public OpRewritePattern<tosa::ResizeOp> {
   }
 };
 
-// TOSA resize with width or height of 1 may be broadcasted to a wider
-// dimension. This is done by materializing a new tosa.resize without
-// the broadcasting behavior, and an explicit broadcast afterwards.
-class MaterializeResizeBroadcast : public OpRewritePattern<tosa::ResizeOp> {
```

Comment:
Why was this removed? Is this something that could be expanded in the commit message for context?

Reply:
After discussed with the original author, I reverted the rewrite pattern. We can have another patch to remove it later. This patch focuses on TOSA v1.0 update.